packages/flutter_test/test/reference_image_test.dart

```diff
@@ -49,18 +49,18 @@ void main() {
   group('fails', () {
     testWidgets('when image sizes do not match', (WidgetTester tester) async {
       expect(
-        await matchesReferenceImage(await createTestImage(50, 50, red)).matchAsync(createTestImage(100, 100, red)),
+        await matchesReferenceImage(await createTestImage(50, 50, red)).matchAsync(await createTestImage(100, 100, red)),
```

Comment:
nit - instead of nesting these like this maybe put them on separate lines?  e.g.

```
final Image red50 = await createTestImage(50, 50, red);
final Image red100 = await createTestImage(100, 100, red);
expect(
  await matchesReferenceImage(red50).matchAsync(red100),
  equals(...)
);
```

for here and below